### PR TITLE
#73: testcase for DEL command

### PR DESCRIPTION
**Pull request by Rits1272**
This PR aims to improve test coverage by adding test cases for `DEL` command

Issue: https://github.com/DiceDB/dice/issues/73

### CODE REVIEW

**Comment by Rits1272** [comment]
@JyotinderSingh @AshwinKul28 please review this PR. TIA!

File: tests/del_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		expected []interface{}
+	}{
+		{
+			name:     "DEL with set key",
+			commands: []string{"SET k1 v1", "DEL k1"},
+			expected: []interface{}{"OK", int64(1)},
+		},
+		{
+			name:     "DEL with multiple keys",
+			commands: []string{"SET k1 v1", "SET k2 v2", "DEL k1 k2"},
```
Comment (JyotinderSingh) [inline tests/del_test.go:25]: Please also validate that the get call returns nil for deleted keys

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]
Thanks for the changes, left a minor comment.

File: tests/del_test.go (hunk 1 above)
Comment (Rits1272) [reply tests/del_test.go]: Resolved, @JyotinderSingh kindly re-review

**Review by JyotinderSingh — APPROVED** [review]
Thanks for improving the test coverage!

**Pull request merged**